test/services/xccdf_report_parser_test.rb

```diff
@@ -107,7 +107,7 @@ class XCCDFReportParserTest < ActiveSupport::TestCase
                      RuleResult.find(rule_results.ids.sample).host.name
         rule_names = RuleResult.where(id: rule_results.ids).map(&:rule)
                                .pluck(:ref_id)
-        assert rule_names.include?(@report_parser.rule_ids.sample)
+        assert rule_names.include?(@report_parser.rule_results.map(&:id).sample)
```

Comment:
Any reason for removing the `rule_ids` method?

Reply:
It wasn't used in any code, just tests. IMO that's not a valid reason to add a method. If the test should be more DRY, a helper method could be added to the test.